app/vmctl/flags.go

```diff
@@ -544,6 +545,11 @@ var (
 			Usage: "Whether to skip TLS certificate verification when connecting to the remote read address",
 			Value: false,
 		},
+		&cli.StringFlag{
```

Comment:
What about `healthPath`? Should we have control over it as well? Does promscale expose this endpoint?

Reply:
Added logic to control health path as well